pages/linux/wall.md

```diff
@@ -1,7 +1,6 @@
 # wall
 
 > Write a message on the terminals of users currently logged in.
-> Only available to super-user.
```

Comment:
Right. @sbrl - You dismissed your review. So, I'm not sure whether you are negative or positive on this. Have a look at #1421 for further context on this PR.